packages/cli/src/commands/validator/register.ts

```diff
@@ -26,6 +30,21 @@ export default class ValidatorRegister extends BaseCommand {
     const validators = await this.kit.contracts.getValidators()
     const accounts = await this.kit.contracts.getAccounts()
 
+    if (!res.flags.yes) {
+      const requirements = await validators.getValidatorLockedGoldRequirements()
+      const duration = requirements.duration.toNumber() * 1000
+      const check = await cli.prompt(
+        `This will lock your gold for ${humanizeDuration(
```

Comment:
Can we replace `your` in this line with how much gold will be locked? I want to avoid confusion if someone thinks this line means all of their gold will be locked.